spec/controllers/organizations_controller_spec.rb

```diff
@@ -5,6 +5,7 @@
 
   let(:organization)  { GitHubFactory.create_owner_classroom_org }
   let(:user)          { organization.users.first                 }
+  let(:student)       { GitHubFactory.create_classroom_student   }
```

Comment:
@tarebyte yes, the student here is the admin of the organization in the VCR record of the test case.